superset-frontend/plugins/plugin-chart-echarts/src/utils/annotation.ts

```diff
@@ -130,3 +134,13 @@ export function extractAnnotationLabels(
 
   return formulaAnnotationLabels.concat(timeseriesAnnotationLabels);
 }
+
+export function getAnnotationData(
+  chartProps: EchartsTimeseriesChartProps | EchartsMixedTimeseriesProps,
+): AnnotationData {
+  const data = chartProps?.queriesData[0]?.annotation_data as AnnotationData;
+  if (!isEmpty(data)) {
+    return data;
+  }
+  return {};
```

Comment:
Nit: ternary seems more readable here

```suggestion
    return isEmpty(data) ? {} : data;
```

I'm also not sure `isEmpty` is needed. If for whatever reason `annotation_data` becomes anything other than a plain object, it probably means there's bugs in the code or corrupted data and we'd need to fix it.

Reply:
@ktmud Thanks, i will look into this case.